makefile

```diff
@@ -355,7 +374,7 @@ clobber: clean
 clean:
 	@echo "makefile: Removing temporary build products"
 	find . -name "*.pyc" -delete
-	sh -c "find . -name \"__pycache__\" |xargs rm -Rf __pycache__"
+	bash -c "find . -name \"__pycache__\" |xargs -n 1 rm -Rf"
```

Comment:
Do we really need bash.  There may be some platforms left over that still have sh as the default shell.

Reply:
We do depend on bash features in several of our .sh scripts, as well as in several places in makefiles (if conditions, pipefail, ...).

bash is the standard shell in Linux. OS-X has it, too. CygWin has it. There is more danger of not having Bourne shell on Linux than not having bash. If you know of any Linux platforms that don't have bash, post them here.

I checked and I did not find any other sh holdouts in pywbem or pywbemtools. If you know of any, post them here.

Reply:
Sounds good to me.  Forget my comment.